permission_handler/pubspec.yaml

```diff
@@ -22,10 +22,10 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.7.0
-  permission_handler_android: ^10.3.0
-  permission_handler_apple: ^9.1.1
+  permission_handler_android: ^10.3.1
+  permission_handler_apple: ^9.1.4
   permission_handler_windows: ^0.1.3
-  permission_handler_platform_interface: ^3.11.0
+  permission_handler_platform_interface: ^3.11.2
```

Comment:
This update is failing the build unnecessary. The updates are all patch updates which are already covered by the current version constraints. 

Currently the build fails because these versions do not exists (other PRs that are still in review). Rolling back the dependency updates will allow this PR to be moved forward without having to rely on the other PRs to be merged and published.

Reply:
@mvanbeusekom I updated the PR, removing the dependency updates. This means that only the example app is updated. Without explicitly stating the new versions of the `platform_interface` and `apple` packages, we should not note the change to the `Bluetooth` permission. I therefore updated the CHANGELOG as well.

How would we communicate with users that use `permission_handler` directly, that Bluetooth has been updated?